### PR TITLE
Refactors vbms code into its own service

### PR DESCRIPTION
**Pull request by amprokop**
Refactors VBMS code out of the appeal repository and into its own service. 

Possible next refactoring steps:
- Change the class methods to instance methods?
- Test more code in appeal repository

### CODE REVIEW

File: app/models/appeal.rb [hunk 1]
```diff
@@ -400,6 +400,10 @@ def bgs
       BGSService.new
     end
 
+    def vbms
```
Comment (aroltsch) [inline app/models/appeal.rb]: I feel as it is not needed.

File: app/services/external_api/vbms_service.rb [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          "VBMS HTTP Error #{status} " \
+          "(#{name}) #{data[:response_body]}"
+        )
+      end
+    end
+  end
+end
+# :nocov:
+
+class VBMSService
```
Comment (aroltsch) [inline app/services/external_api/vbms_service.rb]: It should probably be ExternalApi::VBMSService

File: app/models/appeal.rb (hunk 1 above)
Comment (amprokop) [inline app/models/appeal.rb]: I put this here so we can easily refactor the service into an instance-level service that we use by calling `VBMSService.new`. Make sense?

File: app/services/external_api/vbms_service.rb (hunk 2 above)
Comment (amprokop) [inline app/services/external_api/vbms_service.rb]: word, yeah

**Pull request merged**